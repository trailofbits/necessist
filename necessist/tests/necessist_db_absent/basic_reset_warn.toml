args = ["--root=examples/basic", "--reset", "--deny=database-does-not-exist"]

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 1
