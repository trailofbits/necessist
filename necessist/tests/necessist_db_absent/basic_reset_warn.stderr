Error: No sqlite database found to reset at "[CWD]/examples/basic/necessist.db"; creating new database
